compendium/DeclarativeServices/test/SharedLibraryExceptionTest.cpp

```diff
@@ -0,0 +1,134 @@
+/*=============================================================================
```

Comment:
file name should follow the same naming convention as the other test files, `TestSharedLibraryException.cpp` instead of `SharedLibraryExceptionTest.cpp`

Reply:
changed as suggested